docs/en/compute-edition/31/admin-guide/configure/permissions.adoc

```diff
@@ -78,7 +78,6 @@ https://main\--prisma-cloud-docs-website\--hlxsites.hlx.live/en/compute-edition/
 |===
 
 ifdef::prisma_cloud[]
-Use these link to download the permissions as a text file for the https://docs.paloaltonetworks.com/content/dam/techdocs/en_US/pdf/prisma/prisma-cloud/prerelease/azure-commercial-permissions-security-coverage.txt[Azure Commercial] | https://docs.paloaltonetworks.com/content/dam/techdocs/en_US/pdf/prisma/prisma-cloud/prerelease/azure-government-permissions-security-coverage.txt[Azure Government] environments
```

Comment:
Were these showing in the content? 
they are within a conditional for PCEE and they were not supposed to be included.